etc/packetbeat.yml

```diff
@@ -36,6 +36,26 @@ interfaces:
 
 ############################# Protocols ######################################
 protocols:
+  dns:
+    # Configure the ports where to listen for DNS traffic. You can disable
+    # the DNS protocol by commenting out the list of ports.
+    ports: [53]
```

Comment:
Nothing to change, but I want to make sure I understand how this works. The above means "port 53 on UDP" because the DNS module only registers itself as a UDP protocol. If it would register itself as a TCP one as well, it would mean "port 53 on UDP or TCP", right?